project/src/main/webapp/script.js

```diff
@@ -58,3 +56,9 @@ async function addLogoutUrlToButton() {
   const logoutPrompt = document.getElementById('logout-prompt');
   logoutPrompt.href = logoutUrl;
 }
+
+async function isLoggedIn() {
```

Comment:
same function isLoggedIn() in 2 diff js files (script.js and index.js). Is it good practice to only have one copy but in the HTML files import both scripts, or have a diff copy of each function per js file? @hlz03 @liangqigong

Reply:
Done